input/sources/tev-000095.yaml

```diff
@@ -1,5 +1,17 @@
 source_id: 95
 
+common_name: CTB 37B
```

Comment:
Put `common_name: RX J1713.7-3946`.
This is by far the most common name used for that source (see the title of the paper you're adding here)

I'm not seeing the `CTB 37B` name here:
http://simbad.u-strasbg.fr/simbad/sim-id?Ident=+SNR+G347.3-00.5&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id#lab_ident
and suggest you just remove it, even if it is correct (I didn't look that up).